utils/arrange-json.js

```diff
@@ -0,0 +1,44 @@
+const fs = require("fs");
+const icons = require("./eos-icons.json");
+
+const categoryList = {
+  action: [],
+  alert: [],
+  "artificial intelligence": [],
```

Comment:
this property has quotes while the others don't

Reply:
Can we not just reuse the array we have in src/optionList.json ? this duplication seems unnecessary as this object could be dynamically created rather than hardcoded

Reply:
This is actually a v1 sort of utils, I copied from the website. I plan to generate this list from the eos-icons itself. The optionList.json is generated from this list. Lets keep this as it is, and in a separate PR, I would push one dynamically generated list from the eos-icons itself.

Reply:
Would resolve this in a separate PR, where I would create the list dynamically and remove the created files from git. I didn't do it in this PR as it already had a lot of changes, and I didn't want to confuse you guys with additional steps while testing. It would be better if the converted files are removed from git history. Would like to hear your opinion on this. @cyntss .

Reply:
sounds good.

Reply:
I will create a new issue so we don't forget